package.json

```diff
@@ -62,23 +63,7 @@
   },
   "repository": "github.com/sablier-labs/lockup",
   "scripts": {
-    "build": "forge build",
-    "build:optimized": "FOUNDRY_PROFILE=optimized forge build",
```

Comment:
Even though I like `Just`, I am of the opinion that having these commands here gives us more flexibility over development. For ex. `just --list` prints list of available commands but doesn't provide info on what they will be executing without looking up the other repo.

What if we only move styling related commands to devkit and keep forge related commands in a separate Justfile that can be placed in this repo? This would give us more control over adjusting those commands during development.

<img width="664" alt="Screenshot 2025-05-27 at 11 51 35" src="https://github.com/user-attachments/assets/de5f9ca9-eaf0-43ca-adf0-eb605a632f8e" />

Reply:
Just is super powerful and flexible.

- You can run `just --show RECIPE` to see the recipe's script **without** running it.
- You can run `just --explain RECIPE` to see the recipe's docs **while also** running it.
- You can override all recipes because I set [`allow-duplicate-recipes`](https://just.systems/man/en/settings.html#settings) to `true`. So if you want to quickly prototype something you can either override `forge-check` and the `devkit` recipe will not be run at all, or you can extend it by defining a new recipe called `my-awesome-check`.